spec/models/iiif_playlist_canvas_presenter_spec.rb

```diff
@@ -182,12 +182,12 @@
 
       it "serializes captions as iiif annotations" do
         expect(subject).to all be_a(IIIFManifest::V3::AnnotationContent)
-        expect(subject.length).to eq 2
+        expect(subject.length).to eq 1
       end
 
-      it "includes paths to supplemental and legacy caption files" do
+      it "includes paths to supplemental but NOT legacy caption files" do
         expect(subject.any? { |content| content.body_id =~ /supplemental_files\/#{caption_file.id}\/captions/ }).to eq true
-        expect(subject.any? { |content| content.body_id =~ /master_files\/#{master_file.id}\/captions/ }).to eq true
+        expect(subject.any? { |content| content.body_id =~ /master_files\/#{master_file.id}\/captions/ }).not_to eq true
```

Comment:
```suggestion
        expect(subject.any? { |content| content.body_id =~ /master_files\/#{master_file.id}\/captions/ }).to eq false
```

I don't really feel strongly one way or the other, but would it be better to switch the check to match on false instead of negating true? Whatever the choice, ln 219 should match for consistency.